src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,39 @@
+package duke.task;
+
+/**
+ * Represents a task of type todo. A <code>Todo</code> object contains a description,
+ * a  boolean representing whether or not the task has been done and additional information.
+ */
+public class Todo extends Task{
```

Comment:
Space is missing between Task and the curly bracket. (Sorry I tend to notice these small things)